scripts/generate-components.js

```diff
@@ -132,9 +132,13 @@ import PropTypes from 'prop-types';
 import {omit} from 'ramda';
 
 const ${Component} = (props) => {
+    const dataAttributes = {};
+    if(props.loading_state && props.loading_state.is_loading) {
+        dataAttributes['data-dash-is-loading'] = true;
+    }
+
```

Comment:
It's typical for "flag" attributes to only be present if they are true. This added code in the comp generation makes sure it's only present if truthy.